builder.md

```diff
@@ -67,6 +67,54 @@ These files have the following structure:
       base0D: "dddddd"
       base0E: "eeeeee"
       base0F: "ffffff"
+    overrides:
```

Comment:
Another thought would be sorting this alphabetically so it's a little easier to check options (eg. there's a default-foreground and a default-background)

Reply:
Hmm, I'd be ok with this but currently this information is sorted by `base0*` variable. I've added a comment above each "section" and sorted those sections alphabetically, what do you think of this?

Reply:
Yeah that makes a lot of sense. Happy with that